app/javascript/src/assets/scss/widgets/_conversation-view.scss

```diff
@@ -2,7 +2,7 @@
   @include flex;
   @include flex-direction(column);
 
-  .chat-list__top {
+  .chat-list-top {
```

Comment:
@nisusam We follow BEM for CSS, could you please update the classes to comply with that?
http://getbem.com/introduction/

Reply:
@pranavrajs Updated this PR